pkg/kubelet/kubelet_pods.go

```diff
@@ -1352,7 +1352,13 @@ func (kl *Kubelet) generateAPIPodStatus(pod *v1.Pod, podStatus *kubecontainer.Po
 			s.Phase = pod.Status.Phase
 		}
 	}
-	kl.probeManager.UpdatePodStatus(pod.UID, s)
+	oldContainerReadiness := make(map[string]bool)
+	if p, found := kl.podManager.GetPodByName(pod.Namespace, pod.Name); found{
```

Comment:
It might be better to add a comment that these old container readiness status are from apiserver and explain why we need to use them as pod container readiness status.

Reply:
Sure, thanks!